types/elasticlunr/document_store.d.ts

```diff
@@ -0,0 +1,123 @@
+/*!
+ * elasticlunr.DocumentStore
+ * Copyright (C) @YEAR Wei Song
```

Comment:
If the type definitions aren't under copyright, this comment shouldn't be included. (If they are, we'll have to review the license before merging.)